src/vorpal.ts

```diff
@@ -2,11 +2,15 @@ import { EventEmitter } from 'events';
 import os from 'os';
 
 import chalk from 'chalk';
-import _ from 'lodash';
+import _, {
```

Comment:
Why do take both? :thinking:

Reply:
Destructuring allows tree-shaking. But hope to factor out lodash entirely..!

Reply:
ok, that's temporary